games-strategy/wesnoth/wesnoth-1.18.2.recipe

```diff
@@ -0,0 +1,130 @@
+throne of Wesnoth, of which you are the legitimate heir... step into the \
+boots of a young officer sent to guard a not-so-sleepy frontier outpost... \
+lead a brutal quest to unite the orcish tribes... vanquish a horde of undead \
+warriors unleashed by a foul necromancer, who also happens to have taken your \
+brother hostage... guide a band of elvish survivors in an epic quest to find \
+a new home... get revenge on the orcs by using the foul art of necromancy...
+
+There are at least two hundred unit types, sixteen races, six major factions, \
+and hundreds of years of history. The world of Wesnoth is absolutely huge and \
+only limited by your creativity — make your own custom units, create your own \
+maps, and write your own scenarios or even full-blown campaigns. You can also \
+challenge up to eight friends—or strangers—and fight in epic multiplayer \
+fantasy battles."
+HOMEPAGE="http://wesnoth.org/"
+COPYRIGHT="2003-2024 The Battle for Wesnoth Project"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://sourceforge.net/projects/wesnoth/files/wesnoth-1.18/wesnoth-$portVersion/wesnoth-$portVersion.tar.bz2"
+CHECKSUM_SHA256="5f9f6d6105d48cf614eb06cf72273d62931dcb5d4982d75e851c3fa2fa70553a"
+PATCHES="wesnoth-$portVersion.patchset"
+ADDITIONAL_FILES="wesnoth.rdef.in"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	wesnoth$secondaryArchSuffix
+	cmd:wesnoth
+	cmd:wesnothd
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libboost_filesystem$secondaryArchSuffix
+	lib:libboost_iostreams$secondaryArchSuffix
+	lib:libboost_locale$secondaryArchSuffix
+	lib:libboost_program_options$secondaryArchSuffix
+	lib:libboost_regex$secondaryArchSuffix
+	lib:libboost_system$secondaryArchSuffix
+	lib:libboost_graph$secondaryArchSuffix
+	lib:libboost_coroutine$secondaryArchSuffix
+	lib:libcairo$secondaryArchSuffix
+	lib:libfontconfig$secondaryArchSuffix
+	lib:libpangocairo_1.0$secondaryArchSuffix
+	lib:libSDL2_2.0$secondaryArchSuffix
+	lib:libSDL2_image_2.0$secondaryArchSuffix
+	lib:libSDL2_mixer_2.0$secondaryArchSuffix
+	lib:libvorbisfile$secondaryArchSuffix
+	lib:libcurl$secondaryArchSuffix
+	lib:libdbus_1$secondaryArchSuffix # Optional, for notification support
+	lib:libicui18n$secondaryArchSuffix
+	lib:libicudata$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
+	lib:libharfbuzz$secondaryArchSuffix
+	lib:libglib_2.0$secondaryArchSuffix
+	lib:libcrypto$secondaryArchSuffix
+	lib:libssl$secondaryArchSuffix
+	lib:libgobject_2.0$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	cairo1.18${secondaryArchSuffix}_devel # do not use xcairo
+	devel:libboost_filesystem$secondaryArchSuffix # >= 1.70.0
+	devel:libboost_iostreams$secondaryArchSuffix # >= 1.70.0
+	devel:libboost_locale$secondaryArchSuffix # >= 1.70.0
+	devel:libboost_program_options$secondaryArchSuffix #> = 1.70.0
+	devel:libboost_regex$secondaryArchSuffix # >= 1.70.0
+	devel:libboost_system$secondaryArchSuffix # >= 1.70.0
```

Comment:
Best to add/keep a version string for boost libraries, otherwise we will never know which ones are used, if 1.83.0 works it would be best to use that one (1.85.0 is currently broken due to ICU75 not around anymore).